readme.md

```diff
@@ -53,463 +45,452 @@ html
     └── footer
 ```
 
-
-## The Document Object (5 min)
+## The Document Object (5 minutes / 0:10)
 
 Each web page loaded in the browser has its own `document` object. The `document` interface serves as an entry point to the web page's content. The document is an example of a *host object*--that is, an object provided to Javascript by the browser environment.
 
-## Nodes (5 minutes)
+## Nodes (5 minutes / 0:15)
 
-Everything in the HTML DOM is called a node. Elements are called element nodes, attributes are called attribute nodes, the text inside elements are called text nodes. There are comment nodes. The document itself is called a document node.
+<!-- AM: Is this section necessary? -->
 
```

Comment:
I wouldn't spend too much time here, but I think it's a worthwhile thing for them to know. You have vanilla JS syntax further down that uses the `parentNode` and `childNodes` methods so it's good to tell them what node means.